e2e/process-services-cloud/start-task-custom-app-cloud.e2e.ts

```diff
@@ -129,4 +131,24 @@ describe('Start Task', () => {
         tasksCloudDemoPage.taskListCloudComponent().getDataTable().checkContentIsDisplayed(standaloneTaskName2);
     });
 
+    it('[C297675] Should create a task unassigned when assignee field is empty in Start Task form', () => {
+
+        tasksCloudDemoPage.openNewTaskForm();
+        startTask.addName(standaloneTaskName2);
```

Comment:
standaloneTaskName2 can we find a name that is a characteristic of this task?

please note 1 2 3 or A B C are not good prefix or suffix for names or variable/method/class

Reply:
Updated the names to 'standaloneTaskName' and 'unassignedTaskName' 👍